src/Avalonia.Base/Visual.Composition.cs

```diff
@@ -134,7 +134,7 @@ internal virtual void SynchronizeCompositionProperties()
         // TODO: Introduce a dirty mask like WPF has, so we don't overwrite properties every time
         
         comp.Offset = new (Bounds.Left, Bounds.Top, 0);
-        comp.Size = new (Bounds.Width, Bounds.Height);
+        comp.Size = comp.OpacityMaskSize = new (Bounds.Width, Bounds.Height); //Make sure opacity mask shares the same size.
```

Comment:
Should be nullable, IMO, if we are planning to have it as a public property.

Reply:
Okay y that also works

Reply:
And it should actually be "opacity mask _rect_", I think, since it's valid to have opacity mask to originate at any coordinates.

Reply:
Previously this was just a size without a transform. So where does the rect come from?

Reply:
It will just be `null`. You can use `OpacityMaskRect ?? new Rect(Size)`

Reply:
Looks like I can't use `Vector?` here